src/he/wolf.c

```diff
@@ -101,6 +101,24 @@ int he_internal_write_packet_header(he_conn_t *conn, he_wire_hdr_t *hdr) {
   return HE_SUCCESS;
 }
 
+static he_return_code_t he_internal_conn_outside_write(he_conn_t *conn, uint8_t *packet,
+                                                       size_t length) {
+  if(conn == NULL || packet == NULL) {
+    return HE_ERR_NULL_POINTER;
+  }
+  if(conn->outside_write_cb == NULL && conn->outside_write_ex_cb == NULL) {
+    return HE_ERR_CONF_OUTSIDE_WRITE_CB_NOT_SET;
+  }
+
+  he_return_code_t ret = HE_SUCCESS;
+  if(conn->outside_write_ex_cb) {
```

Comment:
I think it is better to document this behavior in `public/he.h` what will happen if both `conn->outside_write_ex_cb` and `conn->outside_write_cb` are set